Put the list of kafka connectors on the kafka connect landing page (#1378)
aiven/devportal: docs/products/kafka/kafka-connect/howto/request-new-connector.rst

Request a new connector
=======================

> If you know about new and interesting Apache Kafka®ive connectors you'd like us to support, please send us an email to support@Aiven.io about it to help us shaping the future roadmap. We will evaluate the requested connector and might add support for it.

Reviewer: This "interesting Apache Kafka®ive connectors" should be "interesting Apache Kafka® connectors"?

> If you know about new and interesting Apache Kafka®ive connectors you'd like us to support, please send us an email to support@Aiven.io about it to help us shaping the future roadmap. We will evaluate the requested connector and might add support for it.

Reviewer: ```suggestion
If you know about new and interesting Apache Kafka® connectors you'd like us to support, please email us at support@Aiven.io to help us shape the future roadmap. We will evaluate the requested connector and might add support for it.
```

> Aiven evaluation process for new Apache Kafka Connect connectors checks:

Reviewer: ```suggestion
Aiven's evaluation process for new Apache Kafka Connect connectors checks:
```
